pkg/portal/cluster/clusteroperators.go

```diff
@@ -0,0 +1,56 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	configv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type OperatorInformation struct {
+	Name      string                   `json:"name"`
+	Available configv1.ConditionStatus `json:"available"`
+}
+
+type ClusterOperatorsInformation struct {
+	Operators []OperatorInformation `json:"operators"`
+}
+
+func ClusterOperatorsInformationFromClusterDoc(operators *configv1.ClusterOperatorList) *ClusterOperatorsInformation {
```

Comment:
naming nit: Shouldn't "FromClusterDoc" be "FromOperatorList"?

Reply:
yep